stripe/http_client.py

```diff
@@ -51,13 +51,20 @@
 except ImportError:
     urlfetch = None
 
+# proxy support for the pycurl client
+if sys.version_info >= (3, 0):
+    from urllib.parse import urlparse
+else:
+    from urlparse import urlparse
```

Comment:
I think you can just write `from urlparse import urlparse` and `2to3` will automatically adapt the import for Python 3.